### DISCUSSION TITLE
[qmk] key override signature change in latest qmk

### DISCUSSION THREAD

**Discussion by meras** [manna-harbour/miryoku#361] [Q&A] [answered]
Currently the miryoku_qmk fork seems to be behind the main qmk repo. I ran into a few issues when using an external userspace with latest qmk to compile miryoku.

I've not seen this mentioned elsewhere so I'll leave this here if anyone runs into the same issue.

The [2024 August 25](https://docs.qmk.fm/ChangeLog/20240825#key-override-keymap-c-signature-change-24120) qmk release has a breaking change which affects [/users/manna-harbour_miryoku/manna-harbour_miryoku.c#L56-L59](https://github.com/manna-harbour/miryoku_qmk/blob/2c30ed355ba6c98300f49111182e18cd17b17797/users/manna-harbour_miryoku/manna-harbour_miryoku.c#L56-L59)

The fix is simple and provided in the changelong. In the above file change:

    const key_override_t **key_overrides = (const key_override_t *[]){
        &capsword_key_override,
        NULL
    };

to:

    const key_override_t *key_overrides[] = {
        &capsword_key_override
    };

**Comment by manna-harbour** [comment, marked as answer]
Thanks, I'll add this to https://github.com/manna-harbour/miryoku/discussions/287.

**Comment by Zev18** [comment]
Would like to add to this saying that another change is also required now. `IGNORE_MOD_TAP_INTERRUPT` is no longer necessary as it is now the default behavior of mod-tap keys. [/users/manna-harbour_miryoku/config.h line 15](https://github.com/manna-harbour/miryoku_qmk/blob/2c30ed355ba6c98300f49111182e18cd17b17797/users/manna-harbour_miryoku/config.h#L15) needs to be commented out now as well.

```c
// Prevent normal rollover on alphas from accidentally triggering mods.
// #define IGNORE_MOD_TAP_INTERRUPT
```
